lib/aur-search

```diff
@@ -9,63 +9,51 @@ query_type=search
 search_by=name-desc
 sort_key=Name
 
-json_short() {
-    local Name Version NumVotes Maintainer OutOfDate Description
+parse_short() {
+    jq -r --arg key "$1" '[.results[]] | sort_by(.[$key])[] | [
+        .Name         // "-",
+        .Version      // "-",
+        .NumVotes     // "-",
+        .Maintainer   // "-",
+        .OutOfDate    // "-",
+        .Description  // "-"
+        ] | @tsv'
+}
 
-    jq -r --arg key "$1" '[.results[]] | sort_by(.[$key])[] | .Name,
-        .Version,
-        .NumVotes,
-        .Maintainer // "",
-        .OutOfDate  // "",
-        .Description' | while
-    {
-        read -r Name
-        read -r Version
-        read -r NumVotes
-        read -r Maintainer
-        read -r OutOfDate
-        read -r Description
-    }; do
-        [[ "$OutOfDate"  ]] && OutOfDate="(Out-of-date: $(date -d @"$OutOfDate" '+%d %B %Y'))"
-        [[ "$Maintainer" ]] && unset Maintainer
+output_short() {
```

Comment:
unrelated name change

Reply:
Functions `json_short` and `json_long` are split in two stages - parsing PRC results and printing. This is the printing the output part, thus named `output`.